### PR TITLE
[WIP] Signals: Fixing signal system bugs

### PR DESCRIPTION
**Pull request by kuruk-mm**
I repaired some connections most noticeable are

- Right-click PopMenu from SceneTreeDock has no effect when you press an action
- When we add an action on **Project Settings > Input Map**, this print an error saying "_settings_changed" are not a register method.

**Known bug:**

When you create an Animation on the AnimationPlayer, the emit signal "idle_frame" is printing 'CALL_ERROR_TOO_FEW_ARGUMENTS' (with DEBUG_METHODS_ENABLED)

This is because his slot has 6 arguments (1 from p_method + 5 from VARIANT_ARG_LIST). 
```
void call_deferred(const StringName &p_method, VARIANT_ARG_LIST);
```

And our connect has only one bind variable (from AnimationPlayerEditor::_start_onion_skinning())
```
get_tree()->connect("idle_frame", callable_mp((Object *)this, &Object::call_deferred), varray("_prepare_onion_layers_1"));
```

The easy solution is creating a list of 6 Variant with varray (or make_binds) with null data. But I don't think so.

I spend 90% of my time reading code haha. I'll be faster.

### CODE REVIEW

File: editor/editor_visual_profiler.cpp [hunk 1]
```diff
@@ -667,6 +667,7 @@ void EditorVisualProfiler::_combo_changed(int) {
 void EditorVisualProfiler::_bind_methods() {
 
 	ADD_SIGNAL(MethodInfo("enable_profiling", PropertyInfo(Variant::BOOL, "enable")));
+	ADD_SIGNAL(MethodInfo("break_request"));
```
Comment (akien-mga) [inline editor/editor_visual_profiler.cpp:670]: This has been removed several times, it should not be readded :)

It's only relevant for the script profiler, not for the visual and network profilers.

Comment (akien-mga) [reply editor/editor_visual_profiler.cpp:670]: It was removed in a4125bb, maybe it's not in this PR branch. I'll rebase.

Comment (kuruk-mm) [reply editor/editor_visual_profiler.cpp:670]: So we need to delete this connect?
```
visual_profiler->connect("break_request", callable_mp(this, &ScriptEditorDebugger::_profiler_seeked));
```

Comment (akien-mga) [reply editor/editor_visual_profiler.cpp:670]: It was already deleted in a4125bbe, I just rebased to include it. Now you'll have to rebase too.

**Pull request closed without merging**